### PR TITLE
New modules: networkfirewall_policy

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

New module for managing network firewall policy resources.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

networkfirewall_policy
networkfirewall_policy_info

##### ADDITIONAL INFORMATION

### CODE REVIEW

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

File: plugins/module_utils/networkfirewall.py [hunk 1]
```diff
@@ -164,6 +180,73 @@ def _wait_rule_group_active(self, **params):
...
+        update_token = result.get('UpdateToken', None)
+        if update_token:
+            self._update_token = update_token
+        policy = result.get('FirewallPolicy', None)
+        metadata = result.get('FirewallPolicyResponse', None)
+        return dict(FirewallPolicy=policy, FirewallPolicyMetadata=metadata)
+
+    @Boto3Mixin.aws_error_handler('create firewall policy')
+    def _create_policy(self, **params):
+        result = self.client.create_firewall_policy(aws_retry=True, **params)
```
Comment (markuman) [inline plugins/module_utils/networkfirewall.py:212]: wrap into try/except

File: plugins/module_utils/networkfirewall.py [hunk 2]
```diff
@@ -164,6 +180,73 @@ def _wait_rule_group_active(self, **params):
...
+        update_token = result.get('UpdateToken', None)
+        if update_token:
+            self._update_token = update_token
+        return result.get('FirewallPolicyResponse', None)
+
+    @Boto3Mixin.aws_error_handler('update firewall policy')
+    def _update_policy(self, **params):
+        if self._update_token and 'UpdateToken' not in params:
+            params['UpdateToken'] = self._update_token
+        result = self.client.update_firewall_policy(aws_retry=True, **params)
```
Comment (markuman) [inline plugins/module_utils/networkfirewall.py:223]: wrap into try/except

File: plugins/module_utils/networkfirewall.py (hunk 1 above)
Comment (tremble) [reply plugins/module_utils/networkfirewall.py:212]: `@Boto3Mixin.aws_error_handler` is doing this.

Comment (markuman) [reply plugins/module_utils/networkfirewall.py:212]: ah. ok

File: plugins/modules/networkfirewall_policy_info.py [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+  arn:
+    description:
+      - The ARN of the Network Firewall policy.
+    required: false
+    type: str
+  name:
+    description:
+      - The name of the Network Firewall policy.
+    required: false
+    type: str
```
Comment (markuman) [inline plugins/modules/networkfirewall_policy_info.py:25]: some words about mutually exlusive about both parameters can be helpful

**Review by alinabuzachis — APPROVED** [review]

**Review by markuman — APPROVED** [review]

**Pull request merged**